### PR TITLE
remove hive-exec dep from pom.xml

### PR DESCRIPTION
**Pull request by AlvaroVega**

### CODE REVIEW

File: cygnus-ngsi/pom.xml [hunk 1]
```diff
@@ -58,7 +58,14 @@
             <artifactId>cygnus-common</artifactId>
             <version>2.9.0</version>
         </dependency>
-        
+        <!-- https://mvnrepository.com/artifact/org.jodd/jodd-util -->
+        <dependency>
+            <groupId>org.jodd</groupId>
+            <artifactId>jodd-core</artifactId>
+            <version>3.6.8</version>
+        </dependency>
```
Comment (fgalan) [inline cygnus-ngsi/pom.xml:66]: Where this new dependency is used? Java code modifications included in this PR are only deletions...

Comment (AlvaroVega) [reply cygnus-ngsi/pom.xml:66]: Used by https://github.com/telefonicaid/fiware-cygnus/blob/master/cygnus-ngsi/src/main/java/com/telefonica/iot/cygnus/sinks/NGSIArcgisFeatureTableSink.java#L58

Probably imported by hive-exec dep

File: CHANGES_NEXT_RELEASE [hunk 2]
```diff
@@ -0,0 +1 @@
+[cygnus-common] remove hive-exec dependency (unneeded)
```
Comment (fgalan) [inline CHANGES_NEXT_RELEASE:1]:
```suggestion
[cygnus-common] remove hive-exec dependency (unneeded)
[cygnus-ngsi] add jodd-core 3.6.8 dependency (previously included as transitive dependency of hive-exec)
```

**Comment by AlvaroVega** [comment]
to be merged with squash

File: cygnus-ngsi/pom.xml (hunk 1 above)
Comment (fgalan) [reply cygnus-ngsi/pom.xml:66]: NTC

**Review by fgalan — APPROVED** [review]
LGTM

**Pull request merged**